### ISSUE TITLE
New weapon: Thermoshot

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#260]
Warning - warranty void: do not store this weapon with more than 3 shells loaded.

copy from AutoShotgun
reload time 1 second
damage 75%

Special: if this weapon has less than 3 shells in it, it will begin heating up. The damage increases as the heat does, going from base 75%, all the way to 200% in 5 seconds. If however this weapon has more than 3 shells in it, the heat is rapidly dissipated for safety.

An interesting combination of AutoShotgun and Pump, which is all about ammo management.

**Comment by ghost** [comment]
Goddamn, you went on a whole streak of issues. 
Were you like, writing things down the entire month or something?
Besides, what's the point of discussing this, its not like **its gonna be added**, seeing that the developer has less thought of the github anyways.

**Comment by deferred-impact** [comment]
Nah, i had this list of ideas that i never got around to properly formatting on github

> what's the point of discussing this, its not like its gonna be added,

I get your point. But even if most ideas on this page will never be added, I think i gained quite a lot of insight into game design from discussing these. That insight is a surprise tool that will help us later.

**Comment by ghost** [comment]
> That insight is a surprise tool that will help us later.

That will help *you* later.

**Comment by deferred-impact** [comment]
If you don't find discussing ideas useful, then just... don't discuss them

**Comment by ghost** [comment]
No, I was saying that you would need them most in the upcoming time. 
Now I'll assume I don't need to elaborate any further.
